### DISCUSSION TITLE
Date format on Raspberry Pi

### DISCUSSION THREAD

**Discussion by dapper27** [teslamate-org/teslamate#1320] [Q&A]
I am running teslamate on an rpi. I can't find anywhere to change the global settings in grafana to change the date format to anything but the American format! There doesn't seem to be any way to change format settings within grafana itself when on one of the dashboards. Any experience of how to do this?

**Comment by DrMichael** [comment]
I had limited success with adding
```
      - GF_DATE_FORMATS_INTERVAL_HOUR=DD/MM HH:mm
      - GF_DATE_FORMATS_INTERVAL_DAY=DD/MM
      - GF_DATE_FORMATS_FULL_DATE=DD.MMM.YYYY hh:mm:ss
```
to the grafana environment section in docker-compose.yml (it does not work with the 'discrete' panel).

It should also be possible with `GF_USE_BROWSER_LOCALE=true` since version 7.2, but I didn't succeed.

**Reply by dapper27** [reply]
Thanks for the suggestion but it didn't work for me
